README.md

```diff
@@ -2,61 +2,126 @@
+
+## Quickstart
+
+### Prerequisites
 
 1. Install `substreams-sink-sql` from Brew with `brew install streamingfast/tap/substreams-sink-sql` or by using the pre-built binary release [available in the releases page](https://github.com/streamingfast/substreams-sink-sql/releases) (extract `substreams-sink-sql` binary into a folder and ensure this folder is referenced globally via your `PATH` environment variable).
 
-1. Compile the [Substreams](./docs/tutorial/substreams.yaml) tutorial project:
+2. Start Docker Compose in the background:
 
    ```bash
-   cd docs/tutorial
-   cargo build --target wasm32-unknown-unknown --release
-   cd ../..
+   docker compose up -d
    ```
 
-   This creates the following WASM file: `target/wasm32-unknown-unknown/release/substreams_postgresql_sink_tutorial.wasm`
+   > You can wipe the database and restart from scratch by doing `docker compose down` and `rm -rf ./devel/data/postgres`.
 
-1. Start Docker Compose in the background:
+3. Set up environment variables for convenience:
 
    ```bash
-   docker compose up -d
+   export PG_DSN="psql://dev-node:insecure-change-me-in-prod@localhost:5432/dev-node?sslmode=disable"
+   export CLICKHOUSE_DSN="clickhouse://default:default@localhost:9000/default"
    ```
 
-   > You can wipe the database and restart from scratch by doing `docker compose down` and `rm -rf ./devel/data/postgres`.
+   > **Note** To connect to Substreams you will need an authentication token, follow this [guide](https://substreams.streamingfast.io/reference-and-specs/authentication) to obtain one.
+
+## Quickstart Relational Mappings
 
-1. Run the setup command:
+### Postgres
 
-   **Postgres**
+1. Set up the database schema:
 
    ```bash
-   export DSN="postgres://dev-node:insecure-change-me-in-prod@localhost:5432/dev-node?sslmode=disable"
-   substreams-sink-sql setup $DSN docs/tutorial/substreams.yaml
+   substreams-sink-sql setup $PG_DSN solana-spl-token@v0.1.3
    ```
 
-   **Clickhouse**
+2. Run the sink:
 
    ```bash
-   export DSN="clickhouse://default:default@localhost:9000/default"
-   substreams-sink-sql setup $DSN docs/tutorial/substreams.yaml
+   substreams-sink-sql run $PG_DSN solana-spl-token@v0.1.3
    ```
 
-   This will connect to the database and create the schema, using the values from `sink.config.schema`
+### Clickhouse
 
-   > **Note** For the sake of idempotence, we recommend that the schema file only contain `create (...) if not exists` statements.
+1. Set up the database schema:
 
-1. Run the sink
+   ```bash
+   substreams-sink-sql setup $CLICKHOUSE_DSN solana-spl-token@v0.1.3
+   ```
 
-   Now that the code is compiled and the database is set up, let launch the `sink` process.
+2. Run the sink:
 
-   > **Note** To connect to Substreams you will need an authentication token, follow this [guide](https://substreams.streamingfast.io/reference-and-specs/authentication) to obtain one.
+   ```bash
+   substreams-sink-sql run $CLICKHOUSE_DSN solana-spl-token@v0.1.3
+   ```
+
```

Comment:
That is all wrong, those are command for Database Changes only. Please fix the Relational Mappings section by consulting https://github.com/streamingfast/substreams-spl-token?tab=readme-ov-file#quick-start-with-clickhouse to know how to properly run `substreams-sink-sql from-proto`.